release-notes.md

```diff
@@ -106,6 +106,22 @@
 ## Education (zib-Education, nl-core-Education)
 * `.value[x]` ; terminology ; Changed value set from SchooltypeCodelijsten to SchooltypeCBSCodelijst. ; ZIB-1575
 
+## Encounter (zib-Encounter, nl-core-Encounter)
+* Zib Encounter has been refined and several concepts have been moved to the new zib Admission. ; ZIB-1545, ZIB-1104
+* `.status` ; textual ; Refined guidance for DateTime. ; ZIB-1545, ZIB-1104
+* `.class` ; mapping ; Added mapping of EncounterSetting. ; ZIB-1545, ZIB-1104
+* `.type:encounterType` ; slicing ; Added EncounterType to a slice. ; ZIB-1545, ZIB-1104
+* `.type:encounterType` ; terminology ; Changed codes in EncounterTypeCodelist. ; ZIB-1906
+* `.type:reason` ; slicing ; Added mapping of Reason. ; ZIB-1545, ZIB-1104
+* `.type:reason.text` ; mapping ; Moved mapping of CommentEncounterReason from `.reasonCode.extension:commentEncounterReason` and `.reasonReference.extension:commentContactReason`. ; ZIB-1545, ZIB-1104
+* `.participant:responsibleHealthProfessional.individual` ; slicing ; Added ResponsableHealthProfessional to a slice. ; ZIB-1545, ZIB-1104
+* `.period.start` ; textual ; Renamed from StartDateTime to DateTime. ; ZIB-1545, ZIB-1104
+* `.period.end` ; mapping ; Removed mapping of EndDateTime. ; ZIB-1545, ZIB-1104
```

Comment:
Dit gaat niet goed. Als je een Period met alleen een start hebt, dan is dat een periode die nog steeds loopt. De zib lijkt ineens uit te gaan van een moment die geen duur heeft. In FHIR heeft een Encounter altijd juist wel een duur. Dat lijkt me iets om de zib nog op te challengen.

In het uiterste geval zouden we DateTime kunnen mappen naar Period.start én Period.end, met uitleg dat je dan maar dezelfde tijd moet invullen als je het echt niet weet. Maar liever eerst terug naar het zibcentrum. Er wordt nergens gedocumenteerd waarom deze verandering nodig is.